docs/tasks/tools/install-kubectl.md

```diff
@@ -112,6 +112,19 @@ kubectl is available as a [snap](https://snapcraft.io/) application.
 
 2. Run `kubectl version` to verify that the version you've installed is sufficiently up-to-date.
 
+## Install with Powershell from PSGallery
+
+1. If you are on Windows and using [Powershell Gallery](https://https://www.powershellgallery.com/) package manager, you can install and update with:
+
+       Install-Script -Name install-kubectl -Scope CurrentUser -Force     
+       install-kubectl.ps1 [-DownloadLocation <path>]
+       
+If no Downloadlocation is specified, kubectl will be installed in users temp Directory   
+2. The installer creates $HOME/.kube and instucts to create a config file  
+3. Updating
```

Comment:
Can you structure this like the example commands in 1. above - using some wording to explain is before it. For example:

You can update the version of kubectl by rerunning the same scripts again:

    Install-Script -Name install-kubectl -Scope CurrentUser -Force     
    install-kubectl.ps1 [-DownloadLocation <path>]